packages/flutter_tools/test/general.shard/macos/cocoapods_test.dart

```diff
@@ -60,7 +60,13 @@ void main() {
 
   FlutterProject setupProjectUnderTest() {
     // This needs to be run within testWithoutContext and not setUp since FlutterProject uses context.
-    final FlutterProject projectUnderTest = FlutterProject.fromDirectory(fileSystem.directory('project'));
+    final Directory directory = fileSystem.directory('project');
+    directory.childDirectory('ios')
```

Comment:
Since it threw without this, should `defaultHostAppProjectName` instead check if the project exists first if that codepath is possible?